javascript/javascript1/week2/weather.js

```diff
@@ -0,0 +1,12 @@
+function todaysWeather(temperature) {
+  if (temperature <= 7) {
+    return `Please remember to wear winter Jaket.`;
+  } else if (temperature > 7 && temperature < 16) {
+    return `Please wear sweatshirt or autumn jaket.`;
+  } else {
+    return `Hurray!!! Its Summer time, you can wear shorts and shirts.`;
+  }
+}
+
+const whatToWear = todaysWeather(6);
+console.log(whatToWear);
```

Comment:
Can you think of a nested array with temperature and output strings. A for-loop and only a single `if` statement inside ?

Reply:
understood!!!
And made the corrections.